docs/psoc6/tutorial/intro.rst

```diff
@@ -1,4 +1,131 @@
 .. _psoc6_intro:
 
-Getting started with MicroPython on the PSoC6
-=============================================
\ No newline at end of file
+Getting started with MicroPython on the PSoC6™
+==============================================
+
+This tutorial will guide you to get started with running MicroPython on the PSoC6™ microcontrollers. 
+There are only a few step keeping you away from enjoy the python programming experience together
+with the possibilities of PSoC6™ microcontrollers.
+Let's get started!
+
+
+Requirements
+------------
+
+The only required hardware is:
+
+* PSoC6™ board of the :ref:`Supported boards` list.
+* A microUSB cable.
+
+Getting the firmware
+--------------------
+
+The binary *.hex* file is available in the `GitHub release section <https://github.com/jaenrig-ifx/MicroPython/releases>`_. 
+All PSoC6™ firmware versions for each of the supported boards can be found there. 
+
+..
+    .. image:: img/gh-releases.jpg
+..
+    :alt: GitHub MicroPython Releases
+..
+    :width: 360px
+
+Powering the board
+------------------
+
+Connect the USB cable to your computer and the micro USB to the board debugger. All the PSoC6™ boards
+come with a on-board debugger required for flashing/debugging operations during development. Please refer to the
+corresponding board manual of your board.
+
+Deploying the firmware
+----------------------
+
+In order to deploy the firmware in the embedded device, the easiest and quickest way is to use the
+`Cypress Programmer <https://softwaretools.infineon.com/tools/com.ifx.tb.tool.cypressprogrammer>`_.
+It allows to flash a binary in the PSoC6™ microcontrollers family in a few clicks from your Windows,
+Linux or Mac OS machine. 
+Follow the instructions on the provided link to download and install the tool.
+
+After that, select the downloaded MicroPython firmware *.hex* file to be deployed on the PSoC6™. Then, in
+the upper menu, select the connected *Probe/Kit*, click on *Connect*, and finally click on *Program*.
+The log section will show the progress and inform when the firmware deployment on the controller is completed.
+
+.. image:: ../img/cy-programmer.jpg
+    :alt: Cypress Programmer GUI
+    :width: 360px
+
```

Comment:
Just a suggestion: my brain would like the image ceter aligned :) 

```
:align: center
```

Reply:
Might be... but just to keep the same style, as no other pic is aligned center in the docs, let´s keep it like that.
Maybe making a bit bigger helps?

Reply:
Yes, making it a little bit bigger would be better.